python/samples/kernel-syntax-examples/memory_with_docs/data/info1.txt

```diff
@@ -0,0 +1 @@
+Your budget for 2025 is $200,000
```

Comment:
Since this is a kernel example, we don't need 4 separate files that contain one line of text each. You can place the text inside of the kernel example.